src/Greeting.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { cleanup, render, screen } from '@testing-library/react';
+import Greeting from './Greeting';
+
+describe('Input component', () => {
+  afterEach(cleanup);
+
+  it('Greeting render props', () => {
+    const name = 'React';
+    render(<Greeting name={name} />);
```

Comment:
One of the possible variants is a snapshot

`const { container } = render(<Greeting name={name} />);
expect(container).toMatchSnapshot();
`

Reply:
Update in 95e7f1e. An interesting technique. I've never used it before. Thanks!